_data/events/Barcelona-Wallapop.json

```diff
@@ -0,0 +1,17 @@
+{
+  "title": "GDCR Barcelona Wallapop",
+  "url": "https://www.meetup.com/es-ES/Wallapop-Engineering/",
+  "moderators": [
+    "@gerardllorente,@jozaez,@eloipoch"
```

Comment:
Please make this an array of strings so we can link to the individual twitter accounts

Reply:
uhoooo sorry, done!